bin/buttercup

```diff
@@ -83,6 +85,10 @@ do
             shift
             shift
             ;;
+		"--stale-file-error")
```

Comment:
The rest of the file uses spaces, but these lines are using tabs.

Reply:
Thanks.